Dockerfile.builder

```diff
@@ -7,6 +7,4 @@ WORKDIR /app
 # Install packages
 ADD package.json .
 
-RUN npm install
-
-CMD export PATH=./node_modules/.bin:$PATH && npm-watch
+CMD npm install && export PATH=./node_modules/.bin:$PATH && npm-watch
```

Comment:
Why all of this in the cmd?

Reply:
something to do with how RUN doesn't persist into the final container, just cached. If I understand things correctly. This was the only way I could get things to work _every_ time, because of caching containers or something I think?

Reply:
Annoying. Ty for the fix!